clients/cmd/fluent-bit/out_grafana_loki.go

```diff
@@ -53,7 +53,7 @@ func FLBPluginRegister(ctx unsafe.Pointer) int {
 func FLBPluginInit(ctx unsafe.Pointer) int {
 	conf, err := parseConfig(&pluginConfig{ctx: ctx})
 	if err != nil {
-		level.Error(logger).Log("[flb-go]", "failed to launch", "error", err)
+		level.Error(logger).Log("[flb-go]", "failed to launch for out grafana loki config error", "error", err)
```

Comment:
I am not sure if this really improves the error message. I understand the intention though. "failed to launch" could be caused by any a error, so adding a hint that it was caused by parsing the config makes sense.

Maybe something along this line:

```suggestion
		level.Error(logger).Log("[flb-go]", "failed to parse plugin config for out-grafana-loki", "error", err)
```